### DISCUSSION TITLE
Trying to connect to JMAP server with stalwart-cli issue

### DISCUSSION THREAD

**Discussion by SomeKing** [stalwartlabs/jmap-server#49] [Q&A] [answered]
Using a self-signed CA and keep getting a 

`Failed to connect to JMAP server https://*** Transport error: error sending request for url (https://**/.well-known/jmap): error trying to connect: invalid peer certificate contents: invalid peer certificate: UnknownIssuer.`

Confirmed CA is correct by using both CURL to the /auth URI 
Visiting /auth works fine as well. Proper Certificate/CA is being used

Using a Windows CA environment so not sure if a Windows Default is doing something weird.

**Comment by mdecimus** [comment, marked as answer]
Hi,

The CLI does not support self-signed certificates yet, support will be added to the next release in 1/2 months. Could you use a Let's Encrypt certificate for your tests?

**Reply by SomeKing** [reply]
Thanks. Yeah confirmed working with Let's Encrypt Certificate. 
I'll wait for the self-signed support later then.
